MetaMask/metamask-extension
# Added the ability to navigate multiple SIWE notifications

## Explanation

Added the ability to navigate multiple SIWE notifications, now when user sign's multiple SIWE, the navigation header is shown so that he can go through multiple SIWE and also user has option to RejectAll.
* Fixes #17807

## Screenshot
### Before

![multipleSIWENotificationBefore](https://user-images.githubusercontent.com/120601632/224354952-88cbb2d9-b94e-4125-8fc8-c031c4e37a2e.gif)

### After

![multipleSIWENotificationAfter](https://user-images.githubusercontent.com/120601632/224354282-cae12e53-5b56-4742-8b71-7752f688516c.gif)

## Unit tests
### Before

<img width="682" alt="Screenshot 2023-03-10 at 15 35 01" src="https://user-images.githubusercontent.com/120601632/224354362-e1bbd047-4c45-4896-b0da-48a67d281eed.png">

### After

<img width="682" alt="Screenshot 2023-03-10 at 16 17 55" src="https://user-images.githubusercontent.com/120601632/224354374-4c9d84e6-cd64-4324-8378-1e9d9cd28299.png">

## Lint

<img width="682" alt="Screenshot 2023-03-10 at 16 26 14" src="https://user-images.githubusercontent.com/120601632/224355820-e076bc95-3b04-41f6-9281-cf3228c9f01c.png">

## Manual Testing Steps

1. Go to Test Dapp https://metamask.github.io/test-dapp/
2. Connect MetaMask
3. Click to Sign In With Ethereum multiple times
4. In MetaMask extension navigation header is shown and can go through multiple notifications
5. At the bottom of extension Reject requests button is shown
6. When clicked there is pop up modal to Reject
7. After reject extension is closed and there are no requests
